perfscale_regression_ci/utils/cleanup.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+import time
+import subprocess
+import sys
+import json
+
+# Invokes a given command and returns the stdout
+def invoke(command):
```

Comment:
let's use the run in the utils/ocp_utils to keep consistent because it does pretty much the same thing